src/pages/ApiParticulier.js

```diff
@@ -26,6 +30,8 @@ const IntroDescription = () => (
   </div>
 );
 
+// Le demandeur doit décrire le contexte d'usage de l'API
+
```

Comment:
Either put a new line after this comment or remove it, but you shouldn't use both. I'd rather not add the new line if you ask me.